runtime/pkg/duration/duration.go

```diff
@@ -32,11 +32,11 @@ var (
 	}
 	daxOffsetNotations = map[string]StandardDuration{
 		"PP": {},
```

Comment:
What should we do wrt `PP`? Add an explicit check that it is not allowed in time_range functions